hoot-core-test/src/test/cpp/hoot/core/conflate/RubberSheetTest.cpp

```diff
@@ -130,6 +132,8 @@ class RubberSheetTest : public CppUnit::TestFixture
 
   void runSimpleTest()
   {
+    TestUtils::resetEnvironment();
+
```

Comment:
These are the tests that mysteriously failed before this call to `TestTutils::resetEnvironment()` and ultimately `Tgs::Random::seed`